cdap-metadata-spi/src/main/java/io/cdap/cdap/spi/metadata/MetadataConsumerMetrics.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.spi.metadata;
+
+/**
+ * Defines a way to collect provisioner metrics.
+ */
+public interface MetadataConsumerMetrics {
+  
+  /**
+   * Increases the value of the specific metric by delta.
+   *
+   * @param metricName Name of the counter. Use alphanumeric characters in metric names.
+   * @param delta The value to increase by.
+   */
+  void count(String metricName, int delta);
```

Comment:
Can we make this consistent with `io.cdap.cdap.api.metrics.MetricsCollector` increment() and use String, long as arguments ?